web_programming/instagram_pic.py

```diff
@@ -0,0 +1,16 @@
+import requests
+from bs4 import BeautifulSoup
+import datetime
+
+if __name__ == "__main__":
+    url = input("Enter image url: ")
+    print("Downloading image...")
+    req = requests.get(url)
+    soup = BeautifulSoup(req.content, "html.parser")
```

Comment:
```suggestion
    soup = BeautifulSoup(requests.get(url).content, "html.parser")
```
Avoid creating variables that are only used on the next line unless:
1. The lines are already close to the max_line_length of 88 chars per line -- or --
2. The variable name clarifies something that is unclear